packages/connect-ui/src/main.tsx

```diff
@@ -1,11 +1,11 @@
-import { StrictMode } from 'react';
+// import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 
 import { App } from './App.tsx';
 import './index.css';
 
 createRoot(document.getElementById('root') as HTMLElement).render(
-    <StrictMode>
-        <App />
-    </StrictMode>
+    // <StrictMode>
+    <App />
+    // </StrictMode>
```

Comment:
on purpose?

Reply:
yeah I got annoyed after the thousand popups opened twice